### PR TITLE
adds contribute section

### PR DESCRIPTION
**Pull request by bdougie**
# What is this?

I have completed #37 adding the contribute section. 

See it live [here](https://deploy-preview-1--pharmacist-lisa-21114.netlify.com/)

# Feedback

This is an iteration and I am not a designer, but looking for initial feedback. My thought is that there is a lot of "gulp red" on the page, but that will change when #43 is completed and merged. 

## red
I tried using as much existing CSS, but feel free to provide feedback on anything that could be cleaned up. Reference the preview and let me know though 👉🏾 https://deploy-preview-1--pharmacist-lisa-21114.netlify.com/

## copy

I went off the copy on the mock, but if there are any changes you want made, let me know.

screenshot 2017-01-09 10 58 50

### CODE REVIEW

**Comment by bdougie** [comment]
sorry this is premmature

File: index.html [hunk 1]
```diff
@@ -92,6 +92,26 @@ <h4 class="subheading">
...
+    <div class="container-sml">
+      <div class="flex text-center center">
+        <div class="col-10 center">
+          <div class="gulp-image">
+            <img class="logo-img-lg" src="img/gulp-white-logo.svg">
+          </div>
+          <div class="contribute-copy">
+            <h1 class="text-uppercase contribute-item">Become a part of it!</h1>
+            <p class="contribute-item">
+              With your support you'll help us cover all the expenses the collective needs to keep going!
```
Comment (phated) [inline index.html]: This should probably say "all the expenses our project needs"

File: img/gulp-white-logo.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="286px" height="620px" viewBox="0 0 286 620" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
+    <title>gulp-white-logo</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (phated) [inline img/gulp-white-logo.svg]: We weren't sure how `id` works with SVG so we've been removing them.

File: index.html [hunk 3]
```diff
@@ -92,6 +92,26 @@ <h4 class="subheading">
...
 
+  <div class="contribute">
+    <div class="container-sml">
+      <div class="flex text-center center">
+        <div class="col-10 center">
+          <div class="gulp-image">
+            <img class="logo-img-lg" src="img/gulp-white-logo.svg">
+          </div>
+          <div class="contribute-copy">
+            <h1 class="text-uppercase contribute-item">Become a part of it!</h1>
```
Comment (contra) [inline index.html]: "Become a contributor!"

File: index.html [hunk 4]
```diff
@@ -92,6 +92,26 @@ <h4 class="subheading">
...
+            <p class="contribute-item">
+              With your support you'll help us cover all the expenses our project needs to keep going!
```
Comment (contra) [inline index.html]: "With your support, you can help us cover expenses and keep the project going strong."

File: index.html [hunk 5]
```diff
@@ -92,6 +92,26 @@ <h4 class="subheading">
...
+            </p>
+            <a class="ctas-button contribute-item"
+              href="https://opencollective.com/gulpjs">Become a backer or sponsor</a>
```
Comment (contra) [inline index.html]: "Become a backer"

Not really useful to delineate since the page they'll go to has more info.

**Comment by bdougie** [comment]
copy changes made @contra, also thanks for the opportunity 🙌🏾

**Comment by contra** [comment]
@bdougie Thanks! Should be live in a second.

**Pull request merged**